package.json

```diff
@@ -10,13 +10,13 @@
   "license": "SEE LICENSE IN ./LICENSE",
   "scripts": {
     "dev": "NODE_ENV=development webpack-dev-server --hot --progress --https",
+    "dev2": "NODE_ENV=development webpack-dev-server --hot --progress",
```

Comment:
I would like to stick with both. https might be needed for some local feature development. As we have the dev server and the mock server with self signed certificates this makes local testing more annoying. For local development https might be required.

Reply:
Cool, in that case, I would rename `dev2` to `dev:test` so that other contributors not familiar with this PR or the codebase know why we have two ways of running the dev server

Reply:
I still think we should rename one of these, I had to re-read your comment to remind myself what the difference between the two was 😅